Cargo.toml

```diff
@@ -10,3 +10,8 @@ embedded-hal = "0.2.5"
 clap = { version = "4.2.2", features = ["derive"] }
 embedded-hal-mock = "0.9.0"
 linux-embedded-hal = "0.3.2"
+criterion = { version = "0.5.1", features = ["html_reports"] }
```

Comment:
can you move it to be in alphabetic order

Reply:
ok